packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -180,6 +180,10 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 	private readonly isBlobDeleted: (blobPath: string) => boolean;
 	private readonly runtime: IBlobManagerRuntime;
 	private readonly closeContainer: (error?: ICriticalContainerError) => void;
+	private readonly localBlobIdGenerator: () => string;
+	private readonly pendingStashedBlobs: Map<string, Promise<ICreateBlobResponse | void>> =
+		new Map();
+	private stashedBlobsUploadP: Promise<(void | ICreateBlobResponse)[]> | undefined = undefined;
```

Comment:
nit. this could be a lazy promise, which would make the method simpler, and allow this to be immutable (`readonly`)

Reply:
you could also embed the clear into the lazy promise as well

Reply:
I'll take this nit into consideration on a follow up PR